src/appendices/string-method-examples/toUpper-examples.rst

```diff
@@ -0,0 +1,57 @@
+.. _string-touppercase-examples:
+
+``toUpperCase`` Examples
+========================
+
+The general syntax for this method is:
+
+.. sourcecode:: csharp
+
+   stringName.ToUpper();
+
+This method returns a copy of ``stringName`` with all lowercase letters replaced by their uppercase counterparts. It leaves non-alphabetic characters unchanged.
+
+.. admonition:: Example
+
+   .. sourcecode:: csharp
+      :linenos:
+
+      Console.WriteLine("LaunchCode".ToUpper());
+      Console.WriteLine("launchcode".ToUpper());
+      Console.WriteLine("C Sharp rocks!".ToUpper());
+
+   **Output**
+
+   ::
+
+      LAUNCHCODE
+      LAUNCHCODE
+      C SHARP ROCKS!
+
+
+.. admonition:: Example
+
+   When dealing with user input, we can not account for capitalization consistency, 
+   which can cause errors in our code if we are looking for a specfic answer, such as 
+   Using a method like ``ToUpper`` or ``ToLower`` can make 
+
+   This program standardizes an email address by converting it to all lowercase characters.
```

Comment:
this is still a work in progress from my outline7 branch, so i am not sure how it got grouped up into this batch.

Reply:
deleted this, as it's in my other branch too. strangeness.